sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -20,7 +20,7 @@ autorest --java --use=C:/work/autorest.java
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs-pr/main/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json?token=GHSAT0AAAAAABJXDFRPVIKF5L6B4QIZJRVMY3T5YLQ
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs-pr/main/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json?token=GHSAT0AAAAAABJXDFRPV65DHZCSHQ3UQPR6Y6R63CQ
```

Comment:
Now that this library has been released, is there a reason why the swagger is still in private repo?

Reply:
yeah, they just haven't moved it yet. I think its a PR, will follow up to get it merged.